Controller/TranslateController.php

```diff
@@ -72,6 +72,7 @@ public function setSourceLanguage($lang)
      * @Route("/", name="jms_translation_index", options = {"i18n" = false})
      * @Template("@JMSTranslation/Translate/index.html.twig")
```

Comment:
@deguif @goetas I discovered this bug as I'm upgrading from symfony 5..4 to symfony 6.4,

As [SensioFrameworkExtraBundle](https://symfony.com/bundles/SensioFrameworkExtraBundle/current/index.html) It's no longer recommended to used I removed it from my project. Because of the usage of @Template in symfony 6.4 U get the following error when kernel tries to register the route: `In AnnotationException.php line 36:
                                                                                                                                                                                                                          
  [Semantical Error] The annotation "@Sensio\Bundle\FrameworkExtraBundle\Configuration\Template" in method JMS\TranslationBundle\Controller\TranslateController::indexAction() was never imported. Did you maybe forget   
  to add a "use" statement for this annotation?   `
  
  This need to be updated to  `  /**
     * @param Request $request
     *
     * @return Response|array
     *
     * @Route("/", name="jms_translation_index", options = {"i18n" = false})
     */
    #[Route('/', name: 'jms_translation_index', options: ['i18n' => false])]
    #[Template('@JMSTranslation/Translate/index.html.twig')]
    public function indexAction(Request $request)` 
![image](https://github.com/schmittjoh/JMSTranslationBundle/assets/13068685/a5fc904d-df5f-4924-95c9-f1a6ed881c4b)

I'm writing this as I never did contributions to open source and I think it will take me longer to identity the correct way to contribute to this and wait for review as approval.

I thought it would be fixed faster if I ping you guys directly.

Reply:
See https://github.com/schmittjoh/JMSTranslationBundle/pull/569#issuecomment-1419590330 as a possible workaround

Reply:
Thank you